tests/integration/conftest.py

```diff
@@ -18,3 +20,25 @@ def test_env():
     with open(config_file) as f:
         env = yaml.safe_load(f)
     return env
+
+
+@pytest.fixture(scope="session")
+def create_new_request(test_env):
+    """
+    Create a new request in Cachito.
+
+    :param test_env: Test environment configuration
+    :return: Object that contains response from the Cachito API
+    :rtype: Response
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+    response = client.create_new_request(
+        payload={
+            "repo": test_env["package"]["repo"],
+            "ref": test_env["package"]["ref"],
+            "pkg_managers": test_env["package"]["pkg_managers"],
+        },
+    )
+    response = client.wait_for_complete_request(response)
+
+    return response
```

Comment:
Optional: You could return a tuple where the first element is the initial response and second element is completed response. This way, you could use this fixture in `test_creating_new_request`.